schema/message_fmt2/testgen_schema.json

```
@@ -68,14 +68,14 @@
         },
         {
           "properties": {
-            "tests": {
-              "type": "array",
-              "items": {
-                "required": [
-                  "src"
+             "tests": {
+               "type": "array", // For clarity only
```

```suggestion
               "type": "array",
```
Ah, I added this in my other GitHub comment to help explain but we should avoid it in the schema since it's invalid JSON. Sorry for the confusion!

---

+1. The end-to-end CI check is failing unsurprisingly as a result during the initial step to validate the schemas. (FYI @sven-oly schema validation is doing it's job...)

---

Fixed.